other/install_2-1_mysql.sql

```diff
@@ -266,7 +266,8 @@ CREATE TABLE {$db_prefix}log_activity (
   posts SMALLINT UNSIGNED NOT NULL DEFAULT '0',
   registers SMALLINT UNSIGNED NOT NULL DEFAULT '0',
   most_on SMALLINT UNSIGNED NOT NULL DEFAULT '0',
-  PRIMARY KEY (date)
+  PRIMARY KEY (date),
+  INDEX idx_most_on (most_on)
```

Comment:
i find no case in code where searching for most_on,
so useless index?

Reply:
@albertlast It's used for statistical stuff. Sources/Stats.php lines 92, 188, 612 and 721

Reply:
It's also used in Subs_MembersOnline.php.

Reply:
but never in the where statement and never as group by where a index support could be happen -> useless index.

Reply:
@albertlast - wouldn't an index be needed for the MAX(most_on) clauses?  Otherwise full table scans would be needed to find the MAX value, right?

If so, I think it only comes down to how big log_activity gets.  My forum has ~3500 rows, and is ~115K in size.  I believe this is only displayed on the board index.  

Thoughts?   I would lean towards leaving it in given the MAX clause...

Reply:
yes and no
you can use index on max() = yes
but the qry with group by and the other columns should prohibt the index using.

End of the day you can test this by creating the index in you env and use "explain" to look into if the index is used than you know if this realy or not.

Reply:
Removing most_on based on @albertlast 's suggestion.
Looking at the explain plan for the complex queries in question, the index is not used.